src/main/java/frc/utils/battery/BatteryLimiter.java

```diff
@@ -34,10 +61,14 @@ public void execute() {
         double currentAverageVoltage = voltageFilter.calculate(BatteryUtils.getCurrentVoltage());
         if (currentAverageVoltage <= BatteryUtils.MIN_VOLTAGE) {
             reportLowBattery();
-            if (GlobalConstants.ENABLE_BATTERY_LIMITER && !DriverStationUtils.isMatch()) {
+            showBatteryMessage();
+            if (GlobalConstants.ENABLE_BATTERY_LIMITER && !DriverStationUtils.isMatch() && passedMessageStartUpTime()) {
```

Comment:
you check if its a match after you do the popup... you should still state that its low but not with a massive Noam